tests/basic/storage_pool_test.go

```diff
@@ -9993,3 +9994,283 @@ func exitPoolMaintenance(poolId string) {
+			2. Verify after each delete if the right pool got deleted.
+			3. Verify Alerts.
+			4. Try to delete the last pool in the node and it should fail.
+			5. Randomly pick a drive which got freeed up because of pool delete and add it back.
+	*/
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("PoolDelete", "Initiate pool deletion", nil, 0)
+	})
+	var contexts []*scheduler.Context
+
+	ItLog := "Initiate pool delete, then add a new pool and expand the pool"
+	It(ItLog, func() {
+		selectedNode := node.GetStorageNodes()[0]
+		nodePools := selectedNode.StoragePools
+
+		poolToAddBack := nodePools[rand.Intn(len(nodePools))]
+		for _, pool := range nodePools {
+			poolID := strconv.Itoa(int(pool.ID))
+			deletePoolAndValidate(selectedNode, poolID)
+		}
+
+		// verify all pools deleted
+		pools, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+		log.FailOnError(err, "Failed to list storage pools")
+		dash.VerifyFatal(len(pools) == 0, true, "verify all pools deleted")
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("pooldeletefunc-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		stepLog = fmt.Sprintf("Adding cloud drive to node [%v] with size [%v]", selectedNode.Name, poolToAddBack.TotalSize/units.GiB)
+		Step(stepLog, func() {
+			// create spec to perform add drive
+			newSpecSize := poolToAddBack.TotalSize / units.GiB
+			driveSpecs, err := GetCloudDriveDeviceSpecs()
+			log.FailOnError(err, "Error getting cloud drive specs")
+			deviceSpec := driveSpecs[0]
+			deviceSpecParams := strings.Split(deviceSpec, ",")
+			paramsArr := make([]string, 0)
+			for _, param := range deviceSpecParams {
+				if strings.Contains(param, "size") {
+					paramsArr = append(paramsArr, fmt.Sprintf("size=%d,", newSpecSize))
+				} else {
+					paramsArr = append(paramsArr, param)
+				}
+			}
+			newSpec := strings.Join(paramsArr, ",")
+
+			err = Inst().V.AddCloudDrive(&selectedNode, newSpec, -1)
+			log.FailOnError(err, "error adding new drive to node %s", selectedNode.Name)
+			log.InfoD("Validate pool rebalance after drive add to the node %s", selectedNode.Name)
+			err = ValidateDriveRebalance(selectedNode)
+			log.FailOnError(err, "pool re-balance failed on node %s", selectedNode.Name)
+			err = Inst().V.WaitDriverUpOnNode(selectedNode, addDriveUpTimeOut)
+			log.FailOnError(err, "volume drive down on node %s", selectedNode.Name)
+
+			poolsAfr, err := Inst().V.ListStoragePools(metav1.LabelSelector{})
+			log.FailOnError(err, "Failed to list storage pools")
+			dash.VerifyFatal(len(poolsAfr) == 1, true, "verify new pool is created")
+			newPoolsMap, err := Inst().V.GetPoolDrives(&selectedNode)
+			log.FailOnError(err, "error getting pool drive from the node [%s]", selectedNode.Name)
+			dash.VerifyFatal(len(newPoolsMap) == 1, true, "verify new drive is created")
+		})
+
```

Comment:
is L10035: `Verify Alerts.` verified somewhere?

Reply:
No removed that